components/MobileNavigation.js

```diff
@@ -20,11 +19,22 @@ export default class MobileNavigation extends Component {
 
   state = {
     showMobileNav: false,
-    expandMobileNav: false
+    expandMobileNav: false,
+    nowSelectMenu: null
   }
 
   componentDidMount () {
+    const { navs } = this.props
     window.addEventListener('scroll', this.handleScroll)
+    const target = navs.reduce((obj, nav, index) => {
+      const navTarget = {
+        name: nav.label,
+        target: document.getElementById(nav.href.slice(1))
+      }
+      obj[index] = navTarget
+      return obj
+    }, [])
```

Comment:
Since you are reducing into another array keeping the order, this is the same as mapping. Better do this:

```js
    const navigationTargets = navs.map((nav) => {
      return {
        name: nav.label,
        target: document.getElementById(nav.href.slice(1))
      }
    })
```